src/auto-start.ts

```diff
@@ -4,23 +4,27 @@ import autoloader from './plugins/autoloader/prism-autoloader';
 Prism.components.add(autoloader);
 
 export const PrismConfig = {
-	// TODO: Update docs
 	/**
-	 * By default, Prism will attempt to highlight all code elements (by calling {@link Prism#highlightAll}) on the
-	 * current page after the page finished loading. This might be a problem if e.g. you wanted to asynchronously load
+	 * By default, Prism will attempt to highlight all code elements (by calling {@link Prism.highlightAll}) on the
+	 * current page after the page finishes loading. This might be a problem if e.g. you wanted to asynchronously load
 	 * additional languages or plugins yourself.
 	 *
 	 * By setting this value to `true`, Prism will not automatically highlight all code elements on the page.
 	 *
-	 * You obviously have to change this value before the automatic highlighting started. To do this, you can add an
-	 * empty Prism object into the global scope before loading the Prism script like this:
+	 * You have to change this value before the automatic highlighting starts. To do this, you can add an
+	 * empty Prism object into the global scope before the Prism script loads like this:
 	 *
 	 * ```js
 	 * window.Prism = window.Prism || {};
 	 * Prism.manual = true;
 	 * // add a new <script> to load Prism's script
 	 * ```
 	 *
+	 * or you can set `data-manual` on the script tag used to load PrismJs:
+	 * ```html
+	 * <script src="prism.js" data-manual></script>
```

Comment:
It feels like examples of that kind depend on what Prism would look like in v2. I thought it should become ESM-first, no?

Reply:
You can’t get a reference to the script element that triggered loading in ESM. It would need to be an attribute placed anywhere (eg data-prism-manual) or we’d key on the fil name or we’d also support non-ESM imports (by having an entry point that uses dynamic import to load everything else). Or all of the above 😀

Reply:
This is a really good point, I'll try redoing the examples as an actual 3rd party project instead of doing it locally